automated-record-card/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Could we fix the two HTML validation issues flagged when running the content of this document through https://validator.w3.org/nu/#textarea 
1. Consider adding a lang attribute to the html start tag to declare the language of this document.
2. Element head is missing a required instance of child element title.